discord/app/context.py

```diff
@@ -72,6 +76,8 @@ def response(self):
     def respond(self):
         return self.interaction.response.send_message
 
+    reply = respond
```

Comment:
What's the thought process here? What's wrong with `ctx.send`?

Reply:
ctx.send can definitely work just fine. I just wanted to keep that slash command feel where you always get a reply. Plus just like author, reply is familiar where respond is not.